api_blueprint/user.apib

```diff
@@ -127,6 +127,7 @@ Get user data with email, uid or username
             + Default: bar@test.com
         + uid (string, optional)
         + username (string, optional)
+        + active (int, optional)
```

Comment:
for api blueprint, is there a way to specify if a field can be supplied by several different type? (in our case it accepts int, boolean and string right?)

Reply:
@giosakti I've updated my PR